### PR TITLE
pkg/ansible: support multiple workers

### LINKED ISSUE #915: Ansible Operator needs to be able to allow for more than 1 worker per controller
## Bug Report

**What did you do?**
Attempted to have two worker for a specific GVK

**What did you expect to see?**
Have two workers working at the same time.

**What did you see instead? Under which circumstances?**
No way to define two workers.

**Environment**
* operator-sdk version:
0.3.0

* Are you writing your operator in ansible, helm, or go?
ansible

**Possible Solution**
Add option to the watches file and pass to `controller.Options.MaxConcurrentReconciles`. I believe that we will need to add a new way to get the options from Watches and set in the controller options because adding to the interface is not sustainable.

### PR DESCRIPTION
**Pull request by jmrodri**
* add MaxWorkers option
* add support for environment variables
* add WORKER prefix
* add unit test for operator
* update help text
* favor env over cli
* Closes #915

### CODE REVIEW

**Comment by jmrodri** [comment]
With no environment set and no parameter passed in we use the default
```
[jesusr@transam foo-operator{master}]$ unset WORKER_FOOSERVICE_CACHE_EXAMPLE_COM
[jesusr@transam foo-operator{master}]$ operator-sdk up local
INFO[0000] Running the operator locally.                
INFO[0000] Using namespace myoperator.                  
{"level":"info","ts":1551297004.2418368,"logger":"cmd","msg":"Go Version: go1.11.5"}
{"level":"info","ts":1551297004.2418935,"logger":"cmd","msg":"Go OS/Arch: linux/amd64"}
{"level":"info","ts":1551297004.2419198,"logger":"cmd","msg":"Version of operator-sdk: v0.5.0+git"}
{"level":"info","ts":1551297004.2419648,"logger":"cmd","msg":"Watching namespace.","Namespace":"myoperator"}
{"level":"info","ts":1551297004.376555,"logger":"leader","msg":"Trying to become the leader."}
{"level":"info","ts":1551297004.3766854,"logger":"leader","msg":"Skipping leader election; not running in a cluster."}
{"level":"info","ts":1551297004.3778534,"logger":"proxy","msg":"Starting to serve","Address":"127.0.0.1:8888"}
{"level":"dpanic","ts":1551297004.3783045,"logger":"manager","msg":"odd number of arguments passed as key-value pairs for logging","ignored key":1,"stacktrace":"github.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr.handleFields\n\tsrc/github.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr/zapr.go:106\ngithub.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr.(*infoLogger).Info\n\tsrc/github.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr/zapr.go:70\ngithub.com/operator-framework/operator-sdk/pkg/ansible/operator.getMaxWorkers\n\tsrc/github.com/operator-framework/operator-sdk/pkg/ansible/operator/operator.go:88\ngithub.com/operator-framework/operator-sdk/pkg/ansible/operator.Run\n\tsrc/github.com/operator-framework/operator-sdk/pkg/ansible/operator/operator.go:55"}
{"level":"info","ts":1551297004.3782551,"logger":"manager","msg":"using default value for workers %d"}
{"level":"info","ts":1551297004.3784053,"logger":"ansible-controller","msg":"Watching resource","Options.Group":"cache.example.com","Options.Version":"v1alpha1","Options.Kind":"FooService"}
{"level":"info","ts":1551297004.3787527,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"fooservice-controller","source":"kind source: cache.example.com/v1alpha1, Kind=FooService"}
{"level":"info","ts":1551297004.479234,"logger":"kubebuilder.controller","msg":"Starting Controller","controller":"fooservice-controller"}
{"level":"info","ts":1551297004.5796726,"logger":"kubebuilder.controller","msg":"Starting workers","controller":"fooservice-controller","worker count":1}
```

**Comment by jmrodri** [comment]
With no environment set and CLI option passed in with 5 workers we use the options' value
```
[jesusr@transam foo-operator{master}]$ operator-sdk up local --max-workers=5
INFO[0000] Running the operator locally.                
INFO[0000] Using namespace myoperator.                  
{"level":"info","ts":1551297051.8788843,"logger":"cmd","msg":"Go Version: go1.11.5"}
{"level":"info","ts":1551297051.878932,"logger":"cmd","msg":"Go OS/Arch: linux/amd64"}
{"level":"info","ts":1551297051.8789468,"logger":"cmd","msg":"Version of operator-sdk: v0.5.0+git"}
{"level":"info","ts":1551297051.8789768,"logger":"cmd","msg":"Watching namespace.","Namespace":"myoperator"}
{"level":"info","ts":1551297051.990626,"logger":"leader","msg":"Trying to become the leader."}
{"level":"info","ts":1551297051.9907289,"logger":"leader","msg":"Skipping leader election; not running in a cluster."}
{"level":"info","ts":1551297051.991761,"logger":"proxy","msg":"Starting to serve","Address":"127.0.0.1:8888"}
{"level":"dpanic","ts":1551297051.9922717,"logger":"manager","msg":"odd number of arguments passed as key-value pairs for logging","ignored key":5,"stacktrace":"github.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr.handleFields\n\tsrc/github.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr/zapr.go:106\ngithub.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr.(*infoLogger).Info\n\tsrc/github.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr/zapr.go:70\ngithub.com/operator-framework/operator-sdk/pkg/ansible/operator.getMaxWorkers\n\tsrc/github.com/operator-framework/operator-sdk/pkg/ansible/operator/operator.go:88\ngithub.com/operator-framework/operator-sdk/pkg/ansible/operator.Run\n\tsrc/github.com/operator-framework/operator-sdk/pkg/ansible/operator/operator.go:55"}
{"level":"info","ts":1551297051.9921968,"logger":"manager","msg":"using default value for workers %d"}
{"level":"info","ts":1551297051.9923978,"logger":"ansible-controller","msg":"Watching resource","Options.Group":"cache.example.com","Options.Version":"v1alpha1","Options.Kind":"FooService"}
{"level":"info","ts":1551297051.992771,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"fooservice-controller","source":"kind source: cache.example.com/v1alpha1, Kind=FooService"}
{"level":"info","ts":1551297052.0932744,"logger":"kubebuilder.controller","msg":"Starting Controller","controller":"fooservice-controller"}
{"level":"info","ts":1551297052.1936255,"logger":"kubebuilder.controller","msg":"Starting workers","controller":"fooservice-controller","worker count":5}
```

**Comment by jmrodri** [comment]
With the environment set to 3 and no parameter passed in we use the environment:
```
[jesusr@transam foo-operator{master}]$ export WORKER_FOOSERVICE_CACHE_EXAMPLE_COM=3
[jesusr@transam foo-operator{master}]$ operator-sdk up local
INFO[0000] Running the operator locally.                
INFO[0000] Using namespace myoperator.                  
{"level":"info","ts":1551297105.6578155,"logger":"cmd","msg":"Go Version: go1.11.5"}
{"level":"info","ts":1551297105.6578488,"logger":"cmd","msg":"Go OS/Arch: linux/amd64"}
{"level":"info","ts":1551297105.6578596,"logger":"cmd","msg":"Version of operator-sdk: v0.5.0+git"}
{"level":"info","ts":1551297105.657884,"logger":"cmd","msg":"Watching namespace.","Namespace":"myoperator"}
{"level":"info","ts":1551297105.7782464,"logger":"leader","msg":"Trying to become the leader."}
{"level":"info","ts":1551297105.7784433,"logger":"leader","msg":"Skipping leader election; not running in a cluster."}
{"level":"info","ts":1551297105.7822154,"logger":"proxy","msg":"Starting to serve","Address":"127.0.0.1:8888"}
{"level":"info","ts":1551297105.7826939,"logger":"ansible-controller","msg":"Watching resource","Options.Group":"cache.example.com","Options.Version":"v1alpha1","Options.Kind":"FooService"}
{"level":"info","ts":1551297105.7832174,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"fooservice-controller","source":"kind source: cache.example.com/v1alpha1, Kind=FooService"}
{"level":"info","ts":1551297105.8837316,"logger":"kubebuilder.controller","msg":"Starting Controller","controller":"fooservice-controller"}
{"level":"info","ts":1551297105.984061,"logger":"kubebuilder.controller","msg":"Starting workers","controller":"fooservice-controller","worker count":3}
```

**Comment by jmrodri** [comment]
With both the environment set and the parameter passed in we use the ENVIRONMENT. This is counter intuitive but meant for the admin to override what the author may have setup.
```
[jesusr@transam foo-operator{master}]$ export WORKER_FOOSERVICE_CACHE_EXAMPLE_COM=3
[jesusr@transam foo-operator{master}]$ operator-sdk up local --max-workers=5
INFO[0000] Running the operator locally.                
INFO[0000] Using namespace myoperator.                  
{"level":"info","ts":1551297142.418393,"logger":"cmd","msg":"Go Version: go1.11.5"}
{"level":"info","ts":1551297142.418528,"logger":"cmd","msg":"Go OS/Arch: linux/amd64"}
{"level":"info","ts":1551297142.418565,"logger":"cmd","msg":"Version of operator-sdk: v0.5.0+git"}
{"level":"info","ts":1551297142.4188993,"logger":"cmd","msg":"Watching namespace.","Namespace":"myoperator"}
{"level":"info","ts":1551297142.5388122,"logger":"leader","msg":"Trying to become the leader."}
{"level":"info","ts":1551297142.538909,"logger":"leader","msg":"Skipping leader election; not running in a cluster."}
{"level":"info","ts":1551297142.539714,"logger":"proxy","msg":"Starting to serve","Address":"127.0.0.1:8888"}
{"level":"info","ts":1551297142.5400178,"logger":"ansible-controller","msg":"Watching resource","Options.Group":"cache.example.com","Options.Version":"v1alpha1","Options.Kind":"FooService"}
{"level":"info","ts":1551297142.5404913,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"fooservice-controller","source":"kind source: cache.example.com/v1alpha1, Kind=FooService"}
{"level":"info","ts":1551297142.640896,"logger":"kubebuilder.controller","msg":"Starting Controller","controller":"fooservice-controller"}
{"level":"info","ts":1551297142.741315,"logger":"kubebuilder.controller","msg":"Starting workers","controller":"fooservice-controller","worker count":3}

```

**Comment by jmrodri** [comment]
Update help for the parameter to show that the environment will override the value:
```
[jesusr@transam foo-operator{master}]$ operator-sdk run ansible --help
Runs as an ansible operator. This is intended to be used when running
in a Pod inside a cluster. Developers wanting to run their operator locally
should use "up local" instead.

Usage:
  operator-sdk run ansible [flags]

Flags:
  -h, --help                        help for ansible
      --max-workers int             Maximum number of workers to use. Overridden by environment variable. (default 1)
      --reconcile-period duration   Default reconcile period for controllers (default 1m0s)
      --watches-file string         Path to the watches file to use (default "./watches.yaml")
      --zap-devel                   Enable zap development mode (changes defaults to console encoder, debug log level, and disables sampling)
      --zap-encoder encoder         Zap log encoding ('json' or 'console')
      --zap-level level             Zap log level (one of 'debug', 'info', 'error') (default info)
      --zap-sample sample           Enable zap log sampling
```

File: pkg/internal/flags/flags.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ import (
 type WatchFlags struct {
 	ReconcilePeriod time.Duration
 	WatchesFile     string
+	MaxWorkers      int
```
Comment (shawn-hurley) [inline pkg/internal/flags/flags.go]: I think that unless we are adding support for helm in this PR then we need to add this to the ansible specific flags.

File: pkg/ansible/operator/operator.go [hunk 2]
```diff
@@ -43,10 +49,16 @@ func Run(done chan error, mgr manager.Manager, f *flags.AnsibleOperatorFlags, cM
 	c := signals.SetupSignalHandler()
 
 	for gvk, runner := range watches {
+
+		// if the max workers was set on the CLI, use that otherwise, use
```
Comment (shawn-hurley) [inline pkg/ansible/operator/operator.go]: I think this comment is backwards?

File: pkg/ansible/operator/operator.go [hunk 3]
```diff
@@ -67,6 +79,24 @@ func Run(done chan error, mgr manager.Manager, f *flags.AnsibleOperatorFlags, cM
 	done <- mgr.Start(c)
 }
 
+func getMaxWorkers(gvk schema.GroupVersionKind, defvalue int) int {
+	envvar := formatEnvVar(gvk.Kind, gvk.Group)
+	maxWorkers, err := strconv.Atoi(os.Getenv(envvar))
+	if err != nil {
+		// we don't care why we couldn't parse it just use one.
+		// maybe we should log that we are defaulting to 1.
+		logf.Log.WithName("manager").V(0).Info("using default value for workers %d", defvalue)
```
Comment (shawn-hurley) [inline pkg/ansible/operator/operator.go]: I’d say we should probably create a package variable with this and call it manager?

File: pkg/ansible/operator/operator.go (hunk 2 above)
Comment (jmrodri) [reply pkg/ansible/operator/operator.go]: Fixed the comment.

File: pkg/ansible/operator/operator.go (hunk 3 above)
Comment (jmrodri) [reply pkg/ansible/operator/operator.go]: I would generally agree but so far all of the uses of `Log.WithName(...)` use a string literal. I would either leave it alone or in a separate PR update all the uses of `Log.WithName(...)`. Personally I don't think it's a big deal to just have the string literal there.

Here are all the uses in the repo:
```
doc/user/logging.md:  log := logf.Log.WithName("cmd")
doc/user/logging.md:var log = logf.Log.WithName("controller_memcached")
example/memcached-operator/memcached_controller.go.tmpl:var log = logf.Log.WithName("controller_memcached")
pkg/ansible/controller/controller.go:var log = logf.Log.WithName("ansible-controller")
pkg/ansible/controller/reconcile.go:    logger := logf.Log.WithName("reconciler").WithValues(
pkg/ansible/controller/reconcile.go:    logger := logf.Log.WithName("markDone")
pkg/ansible/controller/status/types.go:var log = logf.Log.WithName("controller.status")
pkg/ansible/events/log_events.go:       logger := logf.Log.WithName("logging_event_handler").WithValues(
pkg/ansible/operator/operator.go:               logf.Log.WithName("manager").Error(err, "Failed to get watches")
pkg/ansible/operator/operator.go:               logf.Log.WithName("manager").V(0).Info("using default value for workers %d", defvalue)
pkg/ansible/proxy/kubeconfig/kubeconfig.go:var log = logf.Log.WithName("kubeconfig")
pkg/ansible/proxy/kubectl.go:var log = logf.Log.WithName("proxy")
pkg/ansible/proxy/requestfactory/requestinfo.go:var log = logf.Log.WithName("requestfactory")
pkg/ansible/run.go:var log = logf.Log.WithName("cmd")
pkg/ansible/runner/eventapi/eventapi.go:                logger:     logf.Log.WithName("eventapi").WithValues("job", ident),
pkg/ansible/runner/internal/inputdir/inputdir.go:var log = logf.Log.WithName("inputdir")
pkg/ansible/runner/runner.go:var log = logf.Log.WithName("runner")
pkg/helm/controller/controller.go:var log = logf.Log.WithName("helm.controller")
pkg/helm/run.go:var log = logf.Log.WithName("cmd")
pkg/k8sutil/k8sutil.go:var log = logf.Log.WithName("k8sutil")
pkg/leader/leader.go:var log = logf.Log.WithName("leader")
pkg/metrics/metrics.go:var log = logf.Log.WithName("metrics")
pkg/predicate/predicate.go:var log = logf.Log.WithName("predicate").WithName("eventFilters")
pkg/scaffold/ansible/main.go:           logf.Log.WithName("cmd").Error(err, "")
pkg/scaffold/cmd.go:var log = logf.Log.WithName("cmd")
pkg/scaffold/cmd_test.go:var log = logf.Log.WithName("cmd")
pkg/scaffold/controller_kind.go:var log = logf.Log.WithName("controller_{{ .Resource.LowerKind }}")
pkg/scaffold/controller_kind_test.go:var log = logf.Log.WithName("controller_appservice")
pkg/scaffold/helm/main.go:              logf.Log.WithName("cmd").Error(err, "")
test/test-framework/cmd/manager/main.go:var log = logf.Log.WithName("cmd")
test/test-framework/pkg/controller/memcached/memcached_controller.go:var log = logf.Log.WithName("controller_memcached")
vendor/sigs.k8s.io/controller-runtime/pkg/cache/cache.go:var log = logf.KBLog.WithName("object-cache")
vendor/sigs.k8s.io/controller-runtime/pkg/client/config/config.go:      log                   = logf.KBLog.WithName("client").WithName("config")
vendor/sigs.k8s.io/controller-runtime/pkg/client/fake/client.go:        log = logf.KBLog.WithName("fake-client")
vendor/sigs.k8s.io/controller-runtime/pkg/handler/enqueue.go:var enqueueLog = logf.KBLog.WithName("eventhandler").WithName("EnqueueRequestForObject")
vendor/sigs.k8s.io/controller-runtime/pkg/handler/enqueue_owner.go:var log = logf.KBLog.WithName("eventhandler").WithName("EnqueueRequestForOwner")
vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:var log = logf.KBLog.WithName("controller")
vendor/sigs.k8s.io/controller-runtime/pkg/manager/internal.go:var log = logf.KBLog.WithName("manager")
vendor/sigs.k8s.io/controller-runtime/pkg/predicate/predicate.go:var log = logf.KBLog.WithName("predicate").WithName("eventFilters")
vendor/sigs.k8s.io/controller-runtime/pkg/runtime/log/log.go:   KBLog = Log.WithName("kubebuilder")
vendor/sigs.k8s.io/controller-runtime/pkg/source/internal/eventsource.go:var log = logf.KBLog.WithName("source").WithName("EventHandler")
vendor/sigs.k8s.io/controller-runtime/pkg/source/source.go:var log = logf.KBLog.WithName("source")
vendor/sigs.k8s.io/controller-runtime/pkg/webhook/admission/doc.go:var log = logf.KBLog.WithName("admission")
```

File: pkg/internal/flags/flags.go (hunk 1 above)
Comment (jmrodri) [reply pkg/internal/flags/flags.go]: Done, I moved it to `./pkg/ansible/flags/flag.go`

**Comment by matihost** [comment]
The following logs - when :

ENTRYPOINT ["/usr/local/bin/entrypoint", "--max-workers=10"]


```
{"level":"info","ts":1551800948.7407334,"logger":"proxy","msg":"Starting to serve","Address":"127.0.0.1:8888"}
{"level":"dpanic","ts":1551800948.7419748,"logger":"manager","msg":"odd number of arguments passed as key-value pairs for logging","ignored key":10,"stacktrace":"github.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr.handleFields\n\tsrc/github.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr/zapr.go:106\ngithub.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr.(*infoLogger).Info\n\tsrc/github.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr/zapr.go:70\ngithub.com/operator-framework/operator-sdk/pkg/ansible/operator.getMaxWorkers\n\tsrc/github.com/operator-framework/operator-sdk/pkg/ansible/operator/operator.go:92\ngithub.com/operator-framework/operator-sdk/pkg/ansible/operator.Run\n\tsrc/github.com/operator-framework/operator-sdk/pkg/ansible/operator/operator.go:59"}
{"level":"info","ts":1551800948.7419055,"logger":"manager","msg":"Using default value for workers %d"}
{"level":"info","ts":1551800948.7420409,"logger":"ansible-controller","msg":"Watching resource","Options.Group":"cat.ala.com","Options.Version":"v1","Options.Kind":"AlaIsTheBest"}
{"level":"info","ts":1551800948.7424266,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"AlaIsTheBest-controller","source":"kind source: cat.ala.com/v1, Kind=AlaIsTheBest"}
{"level":"dpanic","ts":1551800948.742684,"logger":"manager","msg":"odd number of arguments passed as key-value pairs for logging","ignored key":10,"stacktrace":"github.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr.handleFields\n\tsrc/github.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr/zapr.go:106\ngithub.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr.(*infoLogger).Info\n\tsrc/github.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr/zapr.go:70\ngithub.com/operator-framework/operator-sdk/pkg/ansible/operator.getMaxWorkers\n\tsrc/github.com/operator-framework/operator-sdk/pkg/ansible/operator/operator.go:92\ngithub.com/operator-framework/operator-sdk/pkg/ansible/operator.Run\n\tsrc/github.com/operator-framework/operator-sdk/pkg/ansible/operator/operator.go:59"}
{"level":"info","ts":1551800948.7426817,"logger":"manager","msg":"Using default value for workers %d"}
{"level":"info","ts":1551800948.742715,"logger":"ansible-controller","msg":"Watching resource","Options.Group":"cat.ala.com","Options.Version":"v1","Options.Kind":"CatCat"}
```

**Comment by matihost** [comment]
The following logs - when :

ENTRYPOINT ["/usr/local/bin/entrypoint", "--max-workers=10"]


```
{"level":"info","ts":1551800948.7407334,"logger":"proxy","msg":"Starting to serve","Address":"127.0.0.1:8888"}
{"level":"dpanic","ts":1551800948.7419748,"logger":"manager","msg":"odd number of arguments passed as key-value pairs for logging","ignored key":10,"stacktrace":"github.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr.handleFields\n\tsrc/github.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr/zapr.go:106\ngithub.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr.(*infoLogger).Info\n\tsrc/github.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr/zapr.go:70\ngithub.com/operator-framework/operator-sdk/pkg/ansible/operator.getMaxWorkers\n\tsrc/github.com/operator-framework/operator-sdk/pkg/ansible/operator/operator.go:92\ngithub.com/operator-framework/operator-sdk/pkg/ansible/operator.Run\n\tsrc/github.com/operator-framework/operator-sdk/pkg/ansible/operator/operator.go:59"}
{"level":"info","ts":1551800948.7419055,"logger":"manager","msg":"Using default value for workers %d"}
{"level":"info","ts":1551800948.7420409,"logger":"ansible-controller","msg":"Watching resource","Options.Group":"cat.ala.com","Options.Version":"v1","Options.Kind":"AlaIsTheBest"}
{"level":"info","ts":1551800948.7424266,"logger":"kubebuilder.controller","msg":"Starting EventSource","controller":"AlaIsTheBest-controller","source":"kind source: cat.ala.com/v1, Kind=AlaIsTheBest"}
{"level":"dpanic","ts":1551800948.742684,"logger":"manager","msg":"odd number of arguments passed as key-value pairs for logging","ignored key":10,"stacktrace":"github.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr.handleFields\n\tsrc/github.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr/zapr.go:106\ngithub.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr.(*infoLogger).Info\n\tsrc/github.com/operator-framework/operator-sdk/vendor/github.com/go-logr/zapr/zapr.go:70\ngithub.com/operator-framework/operator-sdk/pkg/ansible/operator.getMaxWorkers\n\tsrc/github.com/operator-framework/operator-sdk/pkg/ansible/operator/operator.go:92\ngithub.com/operator-framework/operator-sdk/pkg/ansible/operator.Run\n\tsrc/github.com/operator-framework/operator-sdk/pkg/ansible/operator/operator.go:59"}
{"level":"info","ts":1551800948.7426817,"logger":"manager","msg":"Using default value for workers %d"}
{"level":"info","ts":1551800948.742715,"logger":"ansible-controller","msg":"Watching resource","Options.Group":"cat.ala.com","Options.Version":"v1","Options.Kind":"CatCat"}
```

File: pkg/ansible/operator/operator.go [hunk 4]
```diff
@@ -67,6 +83,24 @@ func Run(done chan error, mgr manager.Manager, f *flags.AnsibleOperatorFlags, cM
 	done <- mgr.Start(c)
 }
 
+func getMaxWorkers(gvk schema.GroupVersionKind, defvalue int) int {
+	envvar := formatEnvVar(gvk.Kind, gvk.Group)
+	maxWorkers, err := strconv.Atoi(os.Getenv(envvar))
+	if err != nil {
+		// we don't care why we couldn't parse it just use one.
+		// maybe we should log that we are defaulting to 1.
+		logf.Log.WithName("manager").V(0).Info("Using default value for workers %d", defvalue)
```
Comment (shawn-hurley) [inline pkg/ansible/operator/operator.go]: @matihost it is because of this line @jmrodri can you make a patch please

**Pull request merged**